specification/purview/data-plane/Azure.Analytics.Purview.DataMap/preview/2023-10-01-preview/purviewdatamap.json

```diff
@@ -0,0 +1,7718 @@
+        },
+        "userKey": {
+          "type": "string",
+          "description": "User identity."
+        },
+        "userId": {
+          "type": "string",
+          "description": "User principal name."
+        },
+        "clientIP": {
+          "type": "string",
+          "description": "Client IP address."
+        },
+        "accountId": {
+          "type": "string",
+          "description": "Purview account ID."
+        },
+        "catalogId": {
+          "type": "string",
+          "description": "Purview catalog ID."
+        },
+        "changeRequestId": {
+          "type": "string",
+          "description": "Correlation ID."
+        },
+        "cloudType": {
+          "type": "string",
+          "description": "Cloud type that the object belongs to"
+        },
+        "serviceType": {
+          "type": "string",
+          "description": "Service type that the object belongs to"
+        },
+        "objectId": {
+          "type": "string",
+          "description": "GUID of the targe asset, glossary term or classification definition."
+        },
+        "objectName": {
+          "type": "string",
+          "description": "Object name."
+        },
+        "objectFullyQualifiedName": {
+          "type": "string",
+          "description": "Object fully qualified name."
+        },
+        "objectType": {
+          "type": "string",
+          "description": "Object type name."
+        },
+        "objectCollectionId": {
+          "type": "string",
+          "description": "Object collection ID."
+        },
+        "oldValue": {
+          "type": "string",
+          "description": "Old data value."
+        },
+        "newValue": {
+          "type": "string",
+          "description": "New data value."
+        }
+      },
+      "title": "AuditLog",
+      "description": "Data map audit log."
+    },
+    "EntityLongRunningOperation": {
+      "type": "object",
+      "title": "Status of entity operation",
+      "description": "The status of entity operation",
+      "required": [
+        "id",
+        "status"
+      ],
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "status": {
+          "$ref": "#/definitions/EntityLongRunningOperationStatus"
+        },
+        "createTime": {
+          "type": "string",
+          "description": "The created time of the record."
+        },
+        "lastUpdateTime": {
+          "type": "string",
+          "description": "The last updated time of the record."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/EntityLongRunningOperationProperties"
+        },
+        "error": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AsyncOperationError"
+        }
+      }
+    },
+    "EntityLongRunningOperationProperties": {
+      "type": "object",
+      "additionalProperties": {}
+    },
```

Comment:
Why is it not modeled? It is hard for customer to use without any modeling of the detail.

Reply:
The value will be different for different long running entity operations. The properties here should be a Map<String,String> where the keys will not be the same for different long running operations.

Reply:
Thanks. We will also fix this after we fully onboard with TypeSpec.